bindings/pyroot_experimental/PyROOT/src/PyzCppHelpers.cxx

```diff
@@ -21,11 +21,11 @@ pythonizations.
 #include "CPPInstance.h"
 #include "TClass.h"
 
-PyObject *CallPyObjMethod(PyObject *obj, const char *meth, PyObject *arg1)
+PyObject *CallPyObjMethod(PyObject *obj, const char *meth)
 {
-   // Helper; call method with signature: obj->meth(arg1).
+   // Helper; call method with signature: obj->meth()
    Py_INCREF(obj);
-   PyObject *result = PyObject_CallMethod(obj, const_cast<char *>(meth), const_cast<char *>("O"), arg1);
+   PyObject* result = PyObject_CallMethod(obj, const_cast<char*>(meth), const_cast<char*>(""));
    Py_DECREF(obj);
```

Comment:
Why do we need the increment and decrement of the reference count around the call method?

Reply:
Good point, I don't think we do!